packages/wdio-junit-reporter/src/index.ts

```diff
@@ -222,6 +224,19 @@ class JunitReporter extends WDIOReporter {
             this._fileNameLabel = 'file'
         }
 
+        // there should only be one spec file per runner so we can safely take the first element of the array
+        const specFileName = runner.specs[0]
+        if (isCucumberFrameworkRunner) {
+            builder = this._buildOrderedReport(builder, runner, specFileName, 'feature', isCucumberFrameworkRunner)
+            builder = this._buildOrderedReport(builder, runner, specFileName, 'scenario', isCucumberFrameworkRunner)
```

Comment:
That doesn't make sense to define `builder` twice, if you modify the `builder` parameter in `_buildOrderedReport` you can just leave it as follows:
```suggestion
            this._buildOrderedReport(builder, runner, specFileName, 'feature', isCucumberFrameworkRunner)
            this._buildOrderedReport(builder, runner, specFileName, 'scenario', isCucumberFrameworkRunner)
```